pkg/generate/app/cmd/newapp_test.go

```diff
@@ -53,6 +69,29 @@ func TestValidate(t *testing.T) {
 			env:                 map[string]string{"one": "first", "two": "second", "three": "third"},
 			parms:               map[string]string{},
 		},
+		"envfile1": {
+			cfg: AppConfig{
+				GenerationInputs: GenerationInputs{
+					EnvironmentFile: envFileOperation(envtmpfile, "create"),
+				},
+			},
+			componentValues:     []string{},
+			sourceRepoLocations: []string{},
+			env:                 map[string]string{"one": "first", "two": "second", "three": "third"},
+			parms:               map[string]string{},
+		},
+		"envfile2": {
+			cfg: AppConfig{
+				GenerationInputs: GenerationInputs{
+					EnvironmentFile: envFileOperation(envtmpfile, "create"),
+					Environment:     []string{"one=first1", "two=second", "three=third"},
+				},
+			},
+			componentValues:     []string{},
+			sourceRepoLocations: []string{},
+			env:                 map[string]string{"one": "first1", "two": "second", "three": "third"},
```

Comment:
this test should make sure values provided in the env-file (that are not overridden by the command line args) are present.